### PR TITLE
Display version # and type of the most recent edition on Guide Index

### PR DESCRIPTION
**Pull request by 36degrees**
screen shot 2016-07-28 at 13 59 38

### CODE REVIEW

File: spec/features/guide_index_filter_spec.rb [hunk 1]
```diff
@@ -50,19 +50,19 @@
 
       edition = build(:edition,
                       state: "review_requested",
-                      title: "Edition #{i}",
+                      title: "Guide #{i}",
```
Comment (thehenster) [inline spec/features/guide_index_filter_spec.rb]: :+1:

**Comment by thehenster** [comment]
LGTM

I think I'd prefer extra line to be black though? :)

**Comment by 36degrees** [comment]
screen shot 2016-08-04 at 09 39 04

?

**Comment by thehenster** [comment]
I personally prefer that! Up to you.

Feel free to merge either version :+1:

**Pull request merged**